src/octoprint/static/js/app/viewmodels/files.js

```diff
@@ -501,8 +512,16 @@ $(function() {
             self.uploadSdButton = $("#gcode_upload_sd");
             if (!self.uploadSdButton.length) {
                 self.uploadSdButton = undefined;
+                if (_.endsWith(filename.toLowerCase(), ".stl")) {
```

Comment:
What does this block have to do with warning about a model exceeding the print area? Merge error? Please remove :)